source-pendo/source_pendo/manifest.yaml

```diff
@@ -0,0 +1,164 @@
+version: 0.78.5
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
```

Comment:
I think we can leave the metadata streams included, and have the check still be based on the account info stream. It seems like this would work since the key would default to the row_id.

Reply:
Will do, I'll add them back in.